source/_patterns/01-atoms/dropdown/_dropdown.twig

```diff
@@ -24,15 +27,16 @@
   {% set dropdown_class = 'dropdown' %}
 {% endif %}
 
-{% set classes = [
+{% set dropdown_classes = [
   dropdown_class,
   dropup ? 'dropup',
-] | sort | join(' ') | trim ~ ' ' ~ dropdown_other_classes %}
+  dropdown_other_classes,
```

Comment:
I get why we might specifically scope classes to a thing, like `badge_classes` or `button_classes`, but do we need all these extra undefined(?) `button_other_classes` and `dropdown_other_classes` for anything special? 

twig's `|merge` filter should be fine for passing more classes. it looks like `|merge` is being used for exactly this already in this pr, in the case of action group list items

Reply:
For many of the examples we're not sending in an array to merge and sending a string causes it to kersplode. I can remove the cruft / unused ones for sure, but fwiw I think doing it this way is easier to catch what it's doing at a glance.

Reply:
@tjheffner , I'm going to merge this. Let's make a ticket to address what you brought up.